internal/api/approval/state.go

```diff
@@ -91,6 +99,10 @@ func (s *state) result(log *log.Entry, teams []*github.Team) *Result {
 		// No rules have been matched, which represents an error.
 		result.description = statusEventDescriptionNoRulesMatched
 		result.status = StatusEventStatusPending
+	case len(s.invalidTeamHandles) > 0:
```

Comment:
What happens when both the slice of invalid team and pending team names are not empty? Would GTA still report that there are missing approvals as opposed to that the config is incorrect?

Reply:
Yep, worth ensuring we don't overwrite result when latter case takes precedence over the former. Unless we can easily combine many error case into single string, which may be out of the scope of current improvement.

Reply:
AFAIR there's a character limit for this field, therefore I would vote GTA reports the most important problems first. And in this case misconfiguration seems more important :)

Reply:
@kdrabek-form3 I and @mikolaj-krzyzanowski-f3 refactored this part so that it now sets the status as `error` and skips requesting reviews from the team handles.

We're preparing a functionality that creates a comment with invalid team handles in a separate branch.